src/jit/lsra.cpp

```diff
@@ -1540,6 +1540,7 @@ int LinearScan::compareBlocksForSequencing(BasicBlock* block1, BasicBlock* block
 // Arguments:
 //    sequencedBlockSet - the set of blocks that are already sequenced
 //    block             - the new block to be added
+//    predSet           - the buffer to save predessesors set
```

Comment:
typo: predessesors

I think you should elaborate, e.g.: "a block set allocated by the caller used here as a temporary block set for constructing a predecessor set. Allocated by the caller to avoid reallocating a new block set with every call to this function"